gpushift/meanshift.py

```diff
@@ -22,15 +22,20 @@ def __init__(self, bandwidth, kernel='gaussian', use_keops=True):
         Which kernel to use. Has to be one of MeanShiftStep.KERNELS .
         :param use_keops: bool
         Whether to use the PyKeOps Library or only vanilla PyTorch.
+        :param distance_metric: callable or None
+        If None, uses standard Euclidean distances. For special applications,
+        the passing of a custom distance function is allowed.
```

Comment:
I think it would be good to add the required signature of `distance metric` if it is a callable, including the shapes of the input tensors, to the docstring.